roles/forkliftcontroller/defaults/main.yml

```diff
@@ -28,6 +28,14 @@ controller_container_limits_memory: "800Mi"
 controller_container_requests_cpu: "100m"
 controller_container_requests_memory: "350Mi"
 
+validation_service_name: "{{ app_name }}-validation"
```

Comment:
@pemcg Can we make the validation service a toggable deployment feature similar to feature_ui?  That would allow the user to decide whether to deploy it or not using a boolean on the CR. This requires adding a default feature_validation variable on role defaults and then using it in the main task file with a "when:" clause. It also requires a small update into the ALM examples of the CSV file. There is a sample from feature_ui there as well. I have one question if we should enable the service by default or not, asking @fdupont-redhat for feedback there

Reply:
I've added that as suggested, with the default being on